lib/hooks/api/useFetchMetricStats.ts

```diff
@@ -13,10 +14,11 @@ export type StatsType = {
   forks_count?: number;
 };
 
-export function useFetchMetricStats({ repository, variant, range }: UseFetchMetricStatsParams) {
+export function useFetchMetricStats({ repository, variant, range, orderDirection = "ASC" }: UseFetchMetricStatsParams) {
```

Comment:
The default order is descending in the API. I opted for going ascending here and avoiding reversing the array client-side.